spec/features/listing_a_property_spec.rb

```diff
@@ -0,0 +1,13 @@
+feature "listing a space" do
+  scenario "listing a space on /space/new route" do
+    visit("/spaces/new")
+    #save_and_open_page
+    fill_in("property_name", with: "A beautiful house")
+    fill_in("description", with: "A beautiful house by the river")
```

Comment:
"property_name" etc can be a :symbol instead of a string, but nothing wrong with a string either